service/k8sclient/k8sclient.go

```diff
@@ -923,30 +925,41 @@ func (c *K8sClient) ListPSMDBClusters(ctx context.Context) ([]PSMDBCluster, erro
 	return res, nil
 }
 
+type extraCRParams struct {
+	secretName  string
+	secrets     map[string][]byte
+	psmdbImage  string
+	backupImage string
+	affinity    *psmdb.PodAffinity
+	expose      psmdb.Expose
+	operators   *Operators
+}
+
 // CreatePSMDBCluster creates percona server for mongodb cluster with provided parameters.
 func (c *K8sClient) CreatePSMDBCluster(ctx context.Context, params *PSMDBParams) error {
+	extra := extraCRParams{}
```

Comment:
This can be moved on line 948 since its not needed if there is error during getting cluster

Reply:
:+1: